app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,11 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:id])
```

Comment:
### Optional
- [ ] It would be better to add a model validation to our application so that empty and data not needed will not be saved to the database since it is costly